src/Initializer.php

```diff
@@ -210,4 +214,15 @@ public function initCliOptions(string $cwd, array $argv): CliOptions
         return $cliOptions;
     }
 
+    public function initFormatter(CliOptions $options): FormatterInterface
+    {
+        switch ($options->format) {
+            case 'junit':
+                return new JunitFormatter($this->cwd, $this->printer);
+
+            default:
+                return new ResultFormatter($this->cwd, $this->printer);
```

Comment:
Lets be strict here. 

- empty or 'console' uses the default one
- 'junit' uses the new one
- anything else results in a failure (`InvalidConfigException`).

Reply:
Ideally add simple test to BinTest.